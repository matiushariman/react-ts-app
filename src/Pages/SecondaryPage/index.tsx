import * as React from 'react';

import Heading from 'components/Heading';

export function SecondaryPage(): React.ReactElement {
  return <Heading>SecondaryPage</Heading>;
}

export default SecondaryPage;
