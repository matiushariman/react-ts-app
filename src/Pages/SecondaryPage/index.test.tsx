import React from 'react';
import { cleanup, render } from '@testing-library/react';

import SecondaryPage from '.';

afterEach(cleanup);

test('renders the component', () => {
  const { container } = render(<SecondaryPage />);

  expect(container).toMatchSnapshot();
});
