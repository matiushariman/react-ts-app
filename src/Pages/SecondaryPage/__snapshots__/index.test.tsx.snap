// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div>
  <h1
    class="sc-AxjAm hgJWxN"
  >
    SecondaryPage
  </h1>
</div>
`;
